### PR TITLE
Jetpack DNA: Move Jetpack Sync Defaults from Legacy to psr-4

### PR DESCRIPTION
**Pull request by gititon**
This PR moves Jetpack Sync Defaults from legacy to PSR-4.

No new tests should be needed because this is a refactor, so existing tests should catch any regressions.

### CODE REVIEW

**Comment by roccotripaldi** [comment]
Let's fix the conflicts

File: packages/sync/src/Sender.php [hunk 1]
```diff
@@ -3,6 +3,7 @@
 namespace Automattic\Jetpack\Sync;
 
 use Automattic\Jetpack\Constants;
+use Automattic\Jetpack\Sync\Defaults;
```
Comment (lezama) [inline packages/sync/src/Sender.php]: do we need this when we are "inside" the same namespace?

**Review by roccotripaldi — DISMISSED** [review]
smoke tested

**Review by roccotripaldi — COMMENTED** [review]
Retested after updates!

**Review by roccotripaldi — APPROVED** [review]
hi, this is good. thnkx

**Comment by jeherve** [comment]
Cherry-picked to `branch-7.5` in 7f20484cf959bab5e55f40a811af8c152b628d31

**Pull request merged**